### PR TITLE
Automatic tests

### PR DESCRIPTION
**Pull request by martin-s-a**

### CODE REVIEW

File: autodock/__init__.py [hunk 1]
```diff
@@ -129,16 +129,18 @@ def addAutoDockGPUPackage(cls, env, default=True):
...
-		nvidiaName = cls.getNVIDIAName()
-		compCap = compCapDic[nvidiaName] if nvidiaName in compCapDic else None
-		targetsFlag = f' TARGETS={compCap}' if compCap else ''
-
-		# Installing package
-		installer.getCloneCommand(cls.getAutoDockGPUGithub(), binaryFolderName=cls._atdgpuBinary, targeName='ATDGPU_CLONED')\
-			.addCommand(f'cd {cls._atdgpuBinary} && make DEVICE=GPU OVERLAP=ON{targetsFlag}', 'ATDGPU_COMPILED')\
-			.addPackage(env, dependencies=['git', 'make'], default=default, vars=enVars, updateCuda=True)
+		# Only install AutoDockGPU on systems with Nvidia GPUs
+		if cls.getGPUPlatform() == 'nvidia':
```
Comment (DaniDelHoyo) [inline autodock/__init__.py:133]: I understand that here we could configure it to work with non-nvidia GPUs, but would AutoDockGPU even allow it? If so, we should add the elifs at some point

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
 lxml
 scipion-pyworkflow
 scipion-em
-scipion-chem
\ No newline at end of file
+scipion-chem
+scikit-learn
```
Comment (DaniDelHoyo) [inline requirements.txt:5]: how are we finally going to treat these requirements that are not scipion?

File: autodock/__init__.py (hunk 1 above)
Comment (martin-s-a) [reply autodock/__init__.py:133]: Yes, how to do so is specified [here](https://github.com/ccsb-scripps/AutoDock-GPU/wiki/Guideline-for-users#compilation), but we would need a machine with an AMD GPU or a CPU accelerator card to test the installation, so we know it also works as expected, or if we need also to make some adjustments when running it.

File: requirements.txt (hunk 2 above)
Comment (martin-s-a) [reply requirements.txt:5]: They should be deleted from this file, and, wherever they are being imported and used, move such code to a separate script and call that script with
```Python
protocol.runScript("myscript.py", env="myEnvContainingTheLibrary")
```
For that, the steps would be:

- Create a new enviroment in the installation or reuse an existing one to include such library.
- Move the code to the script and call that script from the protocol or viewer using that env.
- Remove the library dependency from this file.

**Review by DaniDelHoyo — APPROVED** [review]

**Pull request merged**